src/logger.js

```diff
@@ -0,0 +1,26 @@
+import colors from 'colors/safe';
+
+var colorList = ['blue', 'yellow', 'magenta', 'cyan'];
+
+var enabled = !!process.env.DEBUG;
+
+export default function logger (command, ...messages) {
+  if (enabled) {
+    var color = colorList[command.length % colorList.length];
```

Comment:
IMO with colors is more readable. Colors are kinda random but for same command there will be same color

![screen shot 2016-10-10 at 11 21 40 am](https://cloud.githubusercontent.com/assets/26019/19226309/1187b162-8edc-11e6-802e-919f651b2ff8.png)
vs
![screen shot 2016-10-10 at 11 22 46 am](https://cloud.githubusercontent.com/assets/26019/19226295/f367154c-8edb-11e6-9fb9-66adb9dd2559.png)

we can exclude yellow
![screen shot 2016-10-10 at 11 26 00 am](https://cloud.githubusercontent.com/assets/26019/19226330/6670a6de-8edc-11e6-97bb-ed81de83d1f6.png)